src/managers/input/InputManager.cpp

```diff
@@ -530,6 +530,11 @@ void CInputManager::newMouse(wlr_input_device* mouse, bool virt) {
                 libinput_device_config_middle_emulation_set_enabled(LIBINPUTDEV, LIBINPUT_CONFIG_MIDDLE_EMULATION_DISABLED);
         }
 
+        if ((HASCONFIG ? g_pConfigManager->getDeviceInt(PMOUSE->name, "clickfinger_behavior") : g_pConfigManager->getInt("input:touchpad:drag_lock")) == 0)
```

Comment:
wrong

Reply:
ahhhhh, fixed:

Reply:
not fixed, the config for a device called `drag_lock` doesn't exist. add it. you only added the global one